### ISSUE TITLE
[gameobj] Booned critter box showing up as "Other" instead of "Box"

### ISSUE THREAD

**Issue by jeffee83** [elanthia-online/scripts#439]
This usually isn't a problem but I noticed tonight that ;sloot did not pick up this box.

When I picked it up manually and put it in a container I noticed it was not classified as a box:

other (1): an austere deeply-scored coffer swathed in rust-red symbols.

Seems like maybe a new design or something?

Thanks!

**Comment by mrhoribu** [comment]
Do you still have this box? Looking at the gameobj-data.xml it should be marking this correctly as a box. I may need to get my hands on it to check fully.

```xml
^(?:shifting )?((?:(?:acid-pitted|badly damaged|battered|corroded|dented|engraved|enruned|plain|scratched|sturdy) )?(?:brass|gold|iron|mithril|silver|steel) (?:box|chest|coffer|strongbox|trunk)|(?:(?:badly damaged|engraved|enruned|iron-bound|plain|rotting|scratched|simple|sturdy|weathered) )?(?:fel|haon|maoral|modwir|monir|tanik|thanot|wooden) (?:box|chest|coffer|strongbox|trunk)|(?:(?:austere|brass-inlaid|crude|gilded|ornate|scorched) )?(?:carved modwir|cracked|deeply-scored|delicate|red lacquered|stained) (?:box|chest|coffer|strongbox|trunk|case)|filigreed rolaren reliquary)$
```

From the description you have there, I would expect the GameObj for that to show the name as `austere deeply-scored coffer` but will need to hold/play with the XML to see what Lich is seeing the XML as to know why this isn't triggering properly. Also, you are using the latest gameobj-data.xml that released on 2/13?

**Comment by jeffee83** [comment]
;version
gameobj-data last modified....: 2021-02-13 20:25:23 -0500

I have the box and can give it to you in WL/FWI if you'd like. Message me on Zugzugg in game if now's a good time. Thanks!

**Comment by jeffee83** [comment]
Sloot ignores the coffer (it picks up all the other boxes)
```
--- Lich: sloot active.
[sloot]>search #7261128
A fleshless arm thrashes about briefly before decaying into dust.
You search the waern.
Searching carefully, you find an austere deeply-scored coffer swathed in rust-red symbols!
He had nothing else of value.
A glittering waern decays into a compost of fangs and fur.
[You have been awarded 50 Long Term Experience!]
--- Lich: sloot has exited.
```

Coffer showing as other last night:
`In the <a exist="7212752" noun="cloak">black wool cloak</a>:
<pushBold/>weapon,uncommon (2):<popBold/> <a exist="7212802" noun="crowbill">a birch-handled vultite crowbill</a>, <a exist="7212721" noun="staff">a flame-darkened kakore staff cradling an incarnadine orb</a>.
<pushBold/>other (1):<popBold/> <a exist="7274415" noun="coffer">an austere deeply-scored coffer swathed in rust-red symbols</a>.`

Again this morning in my cloak it's Other:
`<pushBold/>other (1):<popBold/> <a exist="11183633" noun="coffer">an austere deeply-scored coffer swathed in rust-red symbols</a>.`

Then I logged in to give it to you - unfortunately I did not look in my cloak first so I have no log of it showing as "Other" again...I took it out and put it back in my cloak and suddenly it was "box" for the first time.
`In the <a exist="11689844" noun="cloak">black wool cloak</a>:
<pushBold/>box (1):<popBold/> <a exist="11689922" noun="coffer">an austere deeply-scored coffer swathed in rust-red symbols</a>.`

**Comment by mrhoribu** [comment]
Fixed via #832
